.gitignore

```diff
@@ -8,10 +8,10 @@ target/
 .idea/jarRepositories.xml
 .idea/compiler.xml
 .idea/libraries/
+/src/main/resources/application.properties
```

Comment:
properties have to be tracked by git, do not exclude them by .gitignore

Reply:
Tracking application.properties. My idea was to hide it, because the db password is stored there.

Reply:
this is just password to your local database. usually this values are stored in some special tools like Vault or Secret Manager AWS and in properties you define them with the help of variables:
```
password=${PASSWORD_VALUE}
```